modules/admin-ui-picard/app/src/thunks/eventDetailsThunks.js

```diff
@@ -0,0 +1,102 @@
+import {
+    loadEventCommentsInProgress,
+    loadEventCommentsSuccess,
+    loadEventCommentsFailure,
+    saveCommentInProgress,
+    saveCommentDone,
+    saveCommentReplyInProgress,
+    saveCommentReplyDone,
+} from '../actions/eventDetailsActions';
+import {
+    getURLParams
+} from "../utils/resourceUtils";
+import axios from "axios";
+
+export const fetchComments = (eventId) => async (dispatch) => {
+    try {
+        dispatch(loadEventCommentsInProgress());
+
+        const commentsData = await axios.get(`admin-ng/event/${eventId}/comments`);
+        const comments = await commentsData.data;
+
+        const commentReasonsData = await axios.get(`admin-ng/resources/components.json`);
+        const commentReasons = (await commentReasonsData.data).eventCommentReasons;
+
+        dispatch(loadEventCommentsSuccess(comments, commentReasons));
+    } catch (e) {
+        dispatch(loadEventCommentsFailure());
+        console.log(e);
+    }
+}
+
+export const saveComment = (eventId, commentText, commentReason) => async (dispatch, getState) => {
+    try {
+        dispatch(saveCommentInProgress());
+
+        const state = getState();
+        let params = getURLParams(state);
+
+        const commentSaved = await axios.post(`admin-ng/event/${eventId}/comment`,
+            `text=${commentText}&reason=${commentReason}`, { params: params });
+        await commentSaved.data;
+
+        dispatch(saveCommentDone());
+        return true;
+    } catch (e) {
+        dispatch(saveCommentDone());
+        console.log(e);
+        return false;
+    }
+}
+
+export const deleteComment = (eventId, commentId) => async (getState) => {
+    try {
+        const state = getState();
+        let params = getURLParams(state);
```

Comment:
Not sure you can use this function because its used for the url params needed in the request for resources displayed in the main table view. Maybe take a look what this function does and define the params you need accordingly.

Reply:
delete doesn't have any parameters, as far as I can see in the old version of the ui, I will remove the params part again from the delete requests